runtime/lua/vim/lsp.lua

```diff
@@ -590,6 +590,9 @@ end
 --- as `initializationOptions`. See `initialize` in the LSP spec.
 ---
 --@param name (string, default=client-id) Name in log messages.
+--
+--@param workspace_folders (table) List of workspace folders passed to the
+--- language server. See `workspaceFolders` in the LSP spec.
```

Comment:
Should the docs mention that this defaults to the root dir if not set?

Reply:
Yes